File: ProjetFinal/serveur/Preferences.js
class Preferences {
    constructor () {
      this.prefers_distance_measured_in_time = true,
      this.prefers_walking = true,
      this.assume_go_back = true,
      this.average_distance = 2
      this.nb_daily_quests = 2
    }

  toJSON () {
    return {
      'prefers_distance_measured_in_time': this.prefers_distance_measured_in_time,
      'prefers_walking': this.prefers_walking,
      'assume_go_back': this.assume_go_back,
      'average_distance': this.average_distance
    };
  }
}
module.exports = Preferences;

File: ProjetFinal/client/app/src/main/java/com/dragonsko/treasurehunt/data/Preferences.kt
package com.dragonsko.treasurehunt.data

data class Preferences(      
    val prefers_distance_measured_in_time: Boolean = true,
                             
    val prefers_walking: Boolean = true,
                             
    val assume_go_back: Boolean = true,
                             
    val average_distance: Int = 1,
    val nb_daily_quests: Int = 2
)
